core-web/libs/sdk/client/src/lib/editor/models/client.model.ts

```diff
@@ -82,3 +98,8 @@ type PostMessageProps<T> = {
 export function postMessageToEditor<T = unknown>(message: PostMessageProps<T>) {
     window.parent.postMessage(message, '*');
 }
+
+export interface DotSDK {
```

Comment:
```suggestion
export interface DotUVE {
```

Because this will hold state of the page when is inside the editor, not the SDK itself.